app/server/routes.ts

```diff
@@ -2,14 +2,18 @@ const express = require('express');
 import * as Paths from 'app/server/paths';
 import { ensureAuthenticated } from 'app/server/middleware/ensure-authenticated';
 import { checkDecision } from 'app/server/middleware/check-decision'
+const config = require('config');
+
+const enableDummyLogin: boolean = config.get('enableDummyLogin');
```

Comment:
These env vars are read in and parsed as strings so ENABLE_DUMMY_LOGIN="false" would result in `config.get('enableDummyLogin')` being true because any non-empty string is true. It's best to do this to be certain
```
config.get('enableDummyLogin') === 'true'
```